<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSJack</class>
 <widget class="QMainWindow" name="CSJack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSJack</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCheckBox" name="sawToggle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>88</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Saw</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pulseToggle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>88</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pulse</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>65</width>
      <height>141</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSlider" name="sineLevel">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Sine Level</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>73</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>90%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="pulseWidth">
     <property name="minimum">
      <number>50</number>
     </property>
     <property name="maximum">
      <number>90</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>50%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Pulse Width</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QDial" name="filterCutoff">
     <property name="value">
      <number>99</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>FilterCutoff</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QDial" name="filterResonance"/>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Resonance</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>547</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
